### ISSUE TITLE
[Feature request] Keep quadratic cover size

### ISSUE THREAD

**Issue by quizzmaster** [PaulWoitaschek/Voice#831]
Since I have my new OnePlus 6T with a much higher screen than my previous Galaxy S7, I noticed that the covers getting stretched in the player view to fit the screen height. This behaviour cuts off the sides so that the covers look really awful. All other music apps, I know, keep the quadratic size of the covers and fill the excessing space otherwise.

My simple suggestion is to keep them quadratic and centering them vertically in the available space.

App-Version: 4.2.0
Phone Name: OnePlus 6T (A6013)
Android Version: 9.0 (Pie)

**Comment by bendmunk95** [comment]
Yes, please! No matter what phone I have, the sides always get cut off. I don't care if the ui is readjusted and gets a little spacing, if just love to see the full cover.

**Comment by Ryuk94** [comment]
I need this so bad, this app is amazing but a qol feature such as this would definitely be appreciated.

**Comment by quizzmaster** [comment]
Any thoughts about this?
